helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -1,11 +1,10 @@
 {{- define "cluster.internal.kubeadm.files" }}
-{{- include "cluster.internal.kubeadm.files.sysctl" . }}
-{{- include "cluster.internal.kubeadm.files.systemd" . }}
-{{- include "cluster.internal.kubeadm.files.ssh" . }}
-{{- include "cluster.internal.kubeadm.files.cri" . }}
-{{- include "cluster.internal.kubeadm.files.kubelet" . }}
-{{- include "cluster.internal.kubeadm.files.kubernetes" . }}
-{{- include "cluster.internal.kubeadm.files.proxy" . }}
+{{- include "cluster.internal.kubeadm.files.sysctl" $ }}
```

Comment:
Minor change, passing the root context here. Currently it doesn't make a difference here, but doing it for the same of consistency with other `include` directives where it's needed.